src/components/cloud-cover/cloud-cover.scss

```diff
@@ -195,6 +197,15 @@ $_gap: ms.step(1);
 
 .c-cloud-cover__content {
   grid-area: content;
+
+  .cloud-cover__description {
```

Comment:
This class name appears to be missing its prefix.

It also appears to be nested, which should not be necessary.